c_cxx/customop/CMakeLists.txt

```diff
@@ -0,0 +1,16 @@
+cmake_minimum_required(VERSION 3.13 FATAL_ERROR)
+project(customop_example C CXX)
+
+set(CMAKE_CXX_STANDARD 17)
+
+option(ONNXRUNTIME_ROOTDIR "onnxruntime root dir")
+
+include_directories("${ONNXRUNTIME_ROOTDIR}/include"
```

Comment:
`include_directories` and `link_directories` is highly discouraged. Please use `target_*` alternative.